docs/user/subprojects.rst

```diff
@@ -1,30 +1,30 @@
-Subprojects
-===========
+Organizing multiple projects in one site: Subprojects
+=====================================================
 
 Projects can be configured in a nested manner, by configuring a project as a
```

Comment:
`nested manner` and `namespace` is not user-friendly language, we should improve this somehow.

Reply:
Yup, this was also missing the "In this article, you can learn about..." softer introduction. I added this, so now there's yet another abstraction layer.